install/install.queries.php

```diff
@@ -704,14 +687,14 @@ function encryptFollowingDefuse($message, $ascii_key)
                             array('admin', 'pwd_default_length', '14'),
                             array('admin', 'tasks_log_retention_delay', '30'),
                             array('admin', 'oauth2_enabled', '0'),
-                            array('admin', 'oauth2_client_id', ''),
-                            array('admin', 'oauth2_client_secret', ''),
+                            array('admin', 'oauth2_client_id', '', '1'),
+                            array('admin', 'oauth2_client_secret', '', '1'),
                             array('admin', 'oauth2_client_endpoint', ''),
-                            array('admin', 'oauth2_client_token', ''),
+                            array('admin', 'oauth2_client_token', '', '1'),
```

Comment:
Missing `oauth2_client_urlResourceOwnerDetails` parameter (seems to have been removed also in the admin settings but it is a used setting).

Reply:
No it is a setting that has been added while provinding generic oauth2 other than EntraID.
So there has never been this setting in admin page.
I will add it

Reply:
Nice, thank you!